vocabularies/voc4cat/0007121.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007121>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007121"^^xsd:token ;
+    skos:altLabel "data channel; signal bin"@en ;
+    skos:definition "A discrete unit or path within a system used to record, process, or represent a portion of a continuous measurement range, often corresponding to a specific interval or signal band."@en ;
```

Comment:
Maybe change "A discrete unit..." to "A discrete signal input..." (I don't understand well what this concept should exactly mean.)

Reply:
Much cleaner def now!